src/lib/git-diff-calculator.js

```diff
@@ -2,20 +2,17 @@
 
 'use strict';
 
-const fs = require( 'fs' );
 const gitDiffRemoteVSHead = require( '../lib/git-diff' );
 const gitDiffIndex = require( '../lib/git-diff-index' );
 
-const config = JSON.parse( fs.readFileSync( '.eslines.json', 'utf-8' ) );
-
-module.exports = function() {
+module.exports = function( remote ) {
```

Comment:
Could the `whatToDiff` option also be a parameter of this function? Let's make the functions in the `lib` directory depend only on their parameters and move all the config-file-reading and environment-variable-checking to another place.